src/components/Filters/FilterTags/FilterTags.js

```diff
@@ -7,6 +7,7 @@ import { Label } from 'semantic-ui-react';
 import { actions as filterActions, selectors as filterSelectors } from '../../../redux/modules/filters';
 import FilterTag from '../FilterTag/FilterTag';
 import { selectors as sources } from '../../../redux/modules/sources';
+import { selectors as tagsSelectors } from '../../../redux/modules/tags';
```

Comment:
Not a good idea as tags is overloaded in this file. For this specific case leaving as is.